client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -92,7 +91,7 @@ export function* createSite(
 	username: string,
 	freeDomainSuggestion?: DomainSuggestion,
 	bearerToken?: string,
-	planSlug?: string
+	isPublicSite = false
```

Comment:
👍

Reply:
Oh, I see we can't take it directly from the Plans store since we are using one of our hooks to determine the active plan.